application/conns.go

```diff
@@ -118,12 +123,21 @@ func (cm *connManager) handleNewConn(regManager *cj.RegistrationManager, clientC
 	}
 	originalDst = originalDstIP.String()
 	flowDescription := fmt.Sprintf("%s -> %s ", originalSrc, originalDst)
-	logger = log.New(os.Stdout, "[CONN] "+flowDescription, golog.Ldate|golog.Lmicroseconds)
+	logger := log.New(os.Stdout, "[CONN] "+flowDescription, golog.Ldate|golog.Lmicroseconds)
+
+	asn, err := regManager.GeoIP.ASN(net.ParseIP(clientConn.RemoteAddr().String()))
```

Comment:
So one thing that comes to mind is that, while all stations currently have access to geoip lookup, it was designed to be optional. So we may want a way to enable / disable the extra lookup. That is we probably shouldn't always assume that the `regmanager.GeoIP` object is not going to be `nil`, because if it is this will cause a panic.

There are several options to fix this,
- if the geoip is nil set some default values for asn / cc and then don't log for those values
- there may be some mechanism in the `registrationManager`  that forces the GeoIP to be non-nil
  - If so maybe we could check the type of the GeoIP when making the stats object.
  - I think the [empty geoip](https://github.com/refraction-networking/conjure/blob/master/application/geoip/empty.go) is used when the dbs are unavailable
- something else? lmk what you think